e2e/webpack/src/webpack.legacy.test.ts

```diff
@@ -113,17 +114,17 @@ describe('Webpack Plugin (legacy)', () => {
     updateFile(
       `${appName}/src/main.ts`,
       `
-      document.querySelector('proj-root').innerHTML = '<h1>Welcome</h1>';
+      document.querySelector('proj-root')!.innerHTML = '<h1>Welcome</h1>';
     `
     );
     updateFile(
       `${appName}/webpack.config.js`,
       `
       const { join } = require('path');
-        const {NxWebpackPlugin} = require('@nx/webpack');
+        const {NxAppWebpackPlugin} = require('@nx/webpack/app-plugin');
         module.exports = {
           output: {
-            path: join(__dirname, '../dist/app9524918'),
```

Comment:
That's weird.. so it always outputted to this random directory?

Reply:
Yep. Which was kinda fine before because we weren't _checking_ the dist folder afterwards, just that the build worked, but with the `e2e` now using static serve, the file-server couldn't find the correct files because the dist folder output didnt match what it was expecting